### PR TITLE
View/WebView with placeholder salt #62

### PR DESCRIPTION
**Pull request by kamarton**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #62

- [x] test

### CODE REVIEW

File: src/WebView.php [hunk 1]
```diff
@@ -164,10 +164,11 @@ public function endPage($ajaxMode = false): void
 
         $content = ob_get_clean();
 
+        var_dump(get_called_class());
```
Comment (samdark) [inline src/WebView.php]: Looks like debug code...

Comment (kamarton) [reply src/WebView.php]: removed

File: src/View.php [hunk 2]
```diff
@@ -120,6 +122,16 @@ public function __construct(string $basePath, Theme $theme, EventDispatcherInter
...
     }
 
+    public function setPlaceholderSalt(string $salt): void
+    {
+        $this->placeholderSalt = $salt;
+    }
+
+    public function getPlaceholderSignature(): string
+    {
+        return md5($this->placeholderSalt);
```
Comment (rustamwin) [inline src/View.php]: What about `sha1`?

Comment (xepozz) [reply src/View.php]: `md5` a little bit faster (~30%)

Comment (rustamwin) [reply src/View.php]: ok

Comment (kamarton) [reply src/View.php]: The algorithm is almost irrelevant, so I changed it to CRC32, which is even faster.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**